internal/ledgerutil/compare.go

```diff
@@ -23,9 +23,14 @@ import (
 	"github.com/pkg/errors"
 )
 
+const (
+	// AllDivsFilename - Filename for the json output that contains all divergences
+	AllDivsFilename = "all_divergences.json"
```

Comment:
Given that the other file will be something like top_divergences_by_height.json, I was thinking this file should be something like all_divergences_by_key.json (since the difference between the files is that this one is sorted by key and the other one is sorted by height).

Reply:
I agree